bin/middlewares/unsupported_browser.js

```diff
@@ -1,5 +1,20 @@
+function iOSversion(userAgent) {
+  if (/iP(hone|od|ad)/.test(userAgent)) {
+    const v = userAgent.match(/OS (\d+)_(\d+)_?(\d+)?/);
+    return [parseInt(v[1], 10), parseInt(v[2], 10), parseInt(v[3] || 0, 10)];
```

Comment:
Beware, this code is not very robust to strange UA strings, which isn't uncommon.
Not a big deal if we flag something as supported when it's not, but if the `.match(…)` fails, `v` will be `null` and the array parse will crash and the page will display a dirty error.
To check for that on local, you have to set the `TILEVIEW_server_unsupportedBrowsers_redirect` env var to `true`, like it's done in prod.

So, we should wrap this fragile code in a `try catch` at least… (but what should be returned?)

Reply:
of course, I used this env var with the value "true" locally, I just didn't push it :) 
thanks for the review, I'm going to make my code less dirty